streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -179,6 +179,44 @@ public void subscribe(final Pattern topics, final ConsumerRebalanceListener list
 
         TestUtils.waitForCondition(() -> assignedTopics.equals(expectedSecondAssignment), STREAM_TASKS_NOT_UPDATED);
 
+        CLUSTER.deleteTopicsAndWait("TEST-TOPIC-1", "TEST-TOPIC-2");
+    }
+
+    @Test
+    public void testRegexRecordsAreProcessedAfterReassignment() throws Exception {
```

Comment:
Thanks for the added tests. Have you verified the test would fail with current trunk?

Also this test takes one minute for me on local, could you take a look and see whether it takes so long?

Reply:
I found reason: here I tried to isolate tests: so we can create same topic in one test, delete it after we are done, do the same in next test, so on. But after each test it calls `streams.close()` in `teardown` method. I guess It tries to do smth with removed topics and gets `TimeoutException` because can't do it. may be it's issue but probably it happens only when close. But it seems to be not related to this task. @abbccdda What do you think?

Reply:
I see, sounds good